docs/source/adapters/site.rst

```diff
@@ -573,7 +573,7 @@ Available machine type configuration options
           MachineMetaData:
             example:
               Cores: 2
-              Memory: 400Mi
+              Memory: 4
```

Comment:
Could you give some further explanations on this, please? What is the current factor here? Naively I would either expect `400` or `0.4` (at least when looking at the previous value of `400Mi`).

Reply:
This is an abritary number that can take any value. Since it is common to have something like 2GB per Core, I set it to 4 GB in this example.

Reply:
Great thanks, I was trying to find a factor from 400 Mi to 4 GB 😁